build/vs_toolchain.py

```diff
@@ -30,7 +30,7 @@
 from gn_helpers import ToGNString
 
 script_dir = os.path.dirname(os.path.realpath(__file__))
-json_data_file = os.path.join(script_dir, 'win_toolchain.json')
+json_data_file = os.path.join(script_dir, 'new_win_toolchain.json')
```

Comment:
No, this file is generated by `gclient sync` via this script. It holds info about the location of the toolchain and the version so that the same version of the toolchain isn't downloaded again.